### ISSUE TITLE
Should the genre column be more consistent?

### ISSUE THREAD

**Issue by samuele-mattiuzzo** [k4m4/movies-for-hackers#236]
This is more of a question with a potential follow-up suggestion: in the genre column, we do have `drama/thriller` and `thriller/drama`

Is there a valid reason behind this? If not, I would suggest making it more consistent (eg: pick one between the two and use it consistently across the list), so that sorting it properly groups them.

Also: since we do have categories already, why the column at all? It seems redundant to me, I would maybe instead replace that with a different rating source maybe (rotten tomatoes?)

**Comment by k4m4** [comment]
I agree with both your suggestions @samuele-mattiuzzo. I'll put this issue up for grabs.

**Comment by samuele-mattiuzzo** [comment]
Happy to take care of this, fancy assigning it to me? I'll clean up all the discrepancies and pick one of either (as I suspect order of tags/categories doesn't truly matter)

**Comment by liam53** [comment]
How do you hack can you tell me

**Comment by liam53** [comment]
How do you hack prodigy sorry for that.
